account_banking_payment_export/models/account_payment.py

```diff
@@ -163,6 +214,58 @@ def action_open(self):
                         "or null (%.2f) !")
                         % (paydict['paylines'][0].partner_id.name,
                            paydict['total']))
-                vals = self._prepare_bank_payment_line(paydict['paylines'])
-                bplo.create(vals)
+                new_values = self._prepare_bank_payment_line(paydict['paylines'])
+                new_values = self.env['bank.payment.line'].\
+                    _add_missing_default_values(new_values)
+
+                new_values.update({
+                    'create_uid': self.env.uid,
+                    'write_uid': self.env.uid,
+                    'create_date': create_time,
+                    'write_date': create_time,
+                    'amount_currency':
+                        sum([payline.amount_currency
+                             for payline in paydict['paylines']])
+                })
+
+                if new_values.get('name', '/') == '/':
+                    new_values['name'] = self.env['ir.sequence'].next_by_code(
+                        'bank.payment.line')
+
+                to_update_payment_line_ids.append(new_values['payment_line_ids'][0][2])
+                del new_values['payment_line_ids']
+
+                all_values.append(new_values)
+
+            created_count = 0
+            created_ids = []
+            for next_values in chunks(all_values, PREFETCH_MAX):
+                created_count += len(next_values)
+                clause, values = get_values_clause_bulk(next_values)
+                self.env.cr.execute("INSERT INTO bank_payment_line %s RETURNING id"
+                                    % (clause), values)
+                _logger.debug("Created %s of %s bank payment lines" %
+                              (created_count, len(all_values)))
```

Comment:
Do we want to have 40000 of such lines in our logs every time?

Reply:
200? for next_values in chunks(all_values, PREFETCH_MAX):

Reply:
I see, thanks!